com.unity.render-pipelines.universal/Editor/UniversalRendererDataEditor.cs

```diff
@@ -18,6 +18,8 @@ private static class Styles
             public static readonly GUIContent TransparentMask = new GUIContent("Transparent Layer Mask", "Controls which transparent layers this renderer draws.");
             public static readonly GUIContent LightingLabel = new GUIContent("Lighting", "Settings related to lighting and rendering paths.");
             public static readonly GUIContent RenderingModeLabel = new GUIContent("Rendering Path", "Select a rendering path.");
+            public static readonly GUIContent DepthPrepassModeLabel = new GUIContent("Depth Prepass Mode", "How this renderer should decide to use a depth prepass. Auto means a depth prepass will be used if needed by a later pass. Forced will always use a depth prepass.");
```

Comment:
Does Unity execute the depth prepass regardless of this setting?
What does "use" mean here? Does the Renderer include something somewhere when it "uses" a depth prepass?
We can have a quick call if the question doesn't make much sense :)

Reply:
I guess "use" should be "execute" here. So it determines whether it's executed as needed or always.